dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -483,34 +486,35 @@ def _get_dataset(value):
+                raise TypeError(error_str + ", got: `{}` of type: {} instead."
+                                .format(input_datasets, type(input_datasets)))
 
             for input_name, input_value in input_datasets.items():
                 # keys are str
                 if not isinstance(input_name, str):
-                    raise TypeError(error_str + (". Expected the keys (representing the input "
-                                                 "names) to be of type `str`, got: `{}` of type: "
-                                                 "{} instead.").format(input_name, type(input_name)))
+                    raise TypeError(error_str +
+                                    (". Expected the keys (representing the input names) to be of "
+                                     "type `str`, got: `{}` of type: {} instead.")
+                                    .format(input_name, type(input_name)))
 
                 # values are tf.data.Dataset or Input
                 is_dataset_only = isinstance(input_value, dataset_ops.DatasetV2)
                 experimental = _get_experimental()
                 if not is_dataset_only and not isinstance(input_value, experimental.Input):
-                    raise TypeError(error_str + (". Expected the values of the dictionary "
-                                                 "(representing the inputs) "
-                                                 " to be of type `tf.data.Dataset` or "
-                                                 "`nvidia.dali.plugin.tf.Input` got: `{}` of "
-                                                 "type: {} instead.").format(input_value, type(input_value)))
+                    raise TypeError(error_str +
+                                    (". Expected the values of the dictionary (representing the "
+                                     "inputs) to be of type `tf.data.Dataset` or "
+                                     "`nvidia.dali.plugin.tf.Input` got: `{}` of type: "
+                                     "{} instead.").format(input_value, type(input_value)))
 
                 # there is External Source with name equal to `input_name`
                 if input_name not in name_es_map.keys():
                     raise ValueError(("Did not find an External Source placeholder node with "
                                       "name='{}' in the provided pipeline - required by the name "
                                       "specified in the `input_datasets`. Names of available  "
                                       "placeholder External Source nodes are: {}. Placeholder "
-                                      "nodes cannot have `source` argument specified.").format(input_name,
-                                                                      list(name_es_map.keys())))
+                                      "nodes cannot have `source` argument specified.")
+                                     .format(input_name, list(name_es_map.keys())))
```

Comment:
May I ask, that you change that to `f""`?

Reply:
```suggestion
                    raise TypeError(error_str +
                                    f". Expected the values of the dictionary (representing the "
                                    f"inputs) to be of type `tf.data.Dataset` or "
                                    f"`nvidia.dali.plugin.tf.Input` got: `{input_value}` of type: "
                                    f"{type(input_value)} instead.")
```